DEI.md

```diff
@@ -0,0 +1,90 @@
+# Diversity, Equity, and Inclusion Project Statement
+
+<!---
+The DEI.md file was originally created in the CHAOSS project. This comment provides attribution of that work as defined 
+under the MIT license. 
+-->
+
+<!---
+Please use the DEI.md Guide at https://github.com/AllInOpenSource/ProjectBadging/blob/main/Guide.DEI.md when creating 
+your DEI.md file
+-->
+
+<img src="https://static.djangoproject.com/img/logos/django-logo-negative.svg" alt="Django" style="width:200px;"/>
+
+Scope of the DEI.md File
+
+Our project prioritizes and reflects on DEI through a regular review of project policies and performance. This 
+reflection is documented in the following DEI.md file based on specific [CHAOSS project](https://chaoss.community) 
+DEI metrics. 
+
+### [Project Access](https://chaoss.community/?p=4953)
+Project access is addressed in our project through various efforts. Through these efforts, we aim to support access for 
+all. Specific efforts in our project include:
+* Our Django Fellows span global time zones and work asynchronously with contributors across the world.
+* The Django documentation and the Django framework itself has multiple translations.
+* All areas of Django are considered open to community contributions, including joining a translation team, conference 
+organising or volunteering, running for the DSF board, Django ticket triage etc.
+* DSF membership, which recognizes contributors to Django, is open to non-code contributors, including contributions 
```

Comment:
I have a worry that we are naturally referring to the `DSF` but the reader (at this point) may have no idea what that means or is. Could we please clarify the first we use the acronym?

Reply:
Ah sorry I didn't mean to remove them all, updated the one above 👍